### DISCUSSION TITLE
zh/config/style

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#58] [Announcements]
# zh/config/style

AList v3的文档

https://alist.nn.ci/zh/config/style.html

**Comment by fengxingsaiwai** [comment]
怎么修改logo？文件放那个目录啊？

**Reply by sobigworm** [reply]
好像不支持把logo放在本地

**Reply by Lipuyari** [reply]
起一个nginx静态资源服务器就好了（很简单的），再把设置好的logo资源链接换上，搞定，记得把设置nginx自启服务

**Comment by chendx-github** [comment]
能不能单独配置每个图标啊  现在的图标全部一个颜色  很难分辨文件夹和文件

**Comment by Soniconce** [comment]
怎么修改logo？能不能用本地文件

**Comment by qychui** [comment]
能默认设置为夜晚模式吗
